### ISSUE TITLE
Multile web roles in Azure Cloud Service

### ISSUE THREAD

**Issue by hommily** [OrchardCMS/Orchard#5352]
Hi,

We used the Orchard.Azure.sln to deploy Orchard to Azure Cloud service with two roles enabled. We configurated the Azure Media Storage, Output cache and db cache correctly according to the documentation at http://docs.orchardproject.net/Documentation/Deploying-Orchard-to-Windows-Azure

The issue we found is that the enabled modules in two roles do not seem to be synchronized (We enabled Module ImportExport in role1, but if we keep hitting F5, if we hit the second web role instance, we can see this module is not listed on the sidebar menu.)

Is this a known issue?

**Comment by Piedone** [comment]
Yes, you'll need to make such change propagate somehow. You can use the built-in Message Bus module on Orchard 1.9+ for this or https://orcharddistributedevents.codeplex.com/ if you're on prior versions.

So this is not an issue but the documentation should be updated.
